recipe/meta.yaml

```diff
@@ -39,6 +39,7 @@ requirements:
   host:
     - python
     - conda-package-handling >=1.3.0
+    - cytoolz >=0.8.1
```

Comment:
Do we need `cytoolz` to build `conda`? Or is it just needed at runtime?

Reply:
During the build we run `conda init --install`

Reply:
Ah I see and so that imports `cytoolz`?

Reply:
https://github.com/conda-forge/conda-feedstock/blob/78cc35ef3614df48de1fab2e3eeb83b0c85fd565/recipe/build.sh#L19

Reply:
yeah see the failing canary builds: https://github.com/conda/conda/runs/7731850991?check_suite_focus=true#step:3:761

Reply:
This isn't required for 4.13 or 4.14, but will be necessary for releases after that